docs/resources/softlayer_virtual_guest.md

```diff
@@ -128,6 +128,10 @@ The following arguments are supported:
     * Provides a primary public IPv6 address.
     * *Optional*
     * *Default*: false
+*   `secondary_ip_count` | *int*
+    * Provides secondary public IPv4 addresses. Acceptable values are 4 and 8. 
+    * *Optional*
+    * *Default*: 0
```

Comment:
I think it is a little confusing saying that the default is 0 when acceptable values are 4 and 8. We should keep that to ourselves, and not say there is any default.